### DISCUSSION TITLE
vcpkg.exe has stopped working

### DISCUSSION THREAD

**Discussion by rebz444** [microsoft/vcpkg#18654] [Q&A] [answered]
I have cMake and visual studio 19 with build tools installed. 

I then pulled and bootstrapped the vcpkg repo. 
> git clone https://github.com/microsoft/vcpkg
> .\vcpkg\bootstrap-vcpkg.bat

I next did 
> .\vcpkg\vcpkg integrate install. 

Then I wanted to run the following command to install packages:
>  .\vcpkg install opencv[ffmpeg]:x64-windows nlopt:x64-windows boost-asio:x64-windows ffmpeg[x264]:x64-windows

but I ran into the error:
> "vcpkg.exe has stopped working"
> "a problem caused the program to stop working correctly.  Windows will close the program and notify you if a solution is available." 
if I select "debug" ->
> "an unhandled win32 exception occurred in [18604] vcpkg.exe. 

Has anyone else run into this? How should I proceed? Help is much appreciated. 
I've attached pictures for my code and the error messages. 

![Screenshot 2021-06-25 151945 (2)](https://user-images.githubusercontent.com/63360362/123475620-a814a480-d5c9-11eb-994d-723ff23443ed.png)
![Screenshot 2021-06-25 152006](https://user-images.githubusercontent.com/63360362/123475636-ad71ef00-d5c9-11eb-9f88-b44816079feb.png)
![Screenshot 2021-06-25 152344](https://user-images.githubusercontent.com/63360362/123475655-b19e0c80-d5c9-11eb-8f5e-33f229549f88.png)

**Comment by autoantwort** [comment, marked as answer]
Looks a little bit like #18636

**Reply by rebz444** [reply]
following what this comment said fixed it.
https://github.com/microsoft/vcpkg/issues/18636#issuecomment-868555508  
vcpkg.exe does NOT work with cmake 3.21.0-rc1, but works properly with 3.20.5.
